styles/index.css

```diff
@@ -15,21 +15,118 @@ body {
 
 h1 {
   text-align: center;
-  border: 1px solid black;
+  border: 1px solid #ee6c4d;
   margin: 0;
   padding: 1.25rem;
   text-transform: uppercase;
 }
 
+/* UTILITY */
+
+.mx-1rem {
+  margin: 0 1rem;
+}
+
+/* COMPONENTS*/
+
+/* QUESTION CARDS*/
+
 .card {
-  border: 1px solid black;
+  border: 1px solid #ee6c4d;
+  padding: 0.5rem;
 }
 
-.navigation {
-  border: 1px solid black;
+.card + .card {
+  margin-top: 1rem;
+}
+
+.questions {
+  width: 90%;
+  margin: 1rem auto;
+  overflow: scroll;
+}
+
+.question {
+  text-align: justify;
+}
+
+.button {
+  display: block;
+  margin: 1rem auto 0.5rem;
+  padding: 0.5rem 1.5rem;
+  background-color: #3d5a80;
+  color: #e0fbfc;
+}
+
+.tagList {
```

Comment:
Please use "kebab-case" instead of "camelCase" for CSS naming (look at property names like `background-color`).